src/skills/doc-detective-validate/SKILL.md

```
@@ -66,7 +66,20 @@ Before completing:
 
 ### Validate Test Specification
 
-Run the validator (try in order; stop and tell the user if none are available):
+**Preferred:** If a tool named `validate_spec` (or `mcp__doc-detective__validate_spec`) is registered in this session, call it instead of shelling out:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)